src/test/java/wolox/training/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,59 @@
+package wolox.training.controllers;
+
+import org.junit.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import wolox.training.models.User;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import static org.mockito.BDDMockito.given;
+
+import static org.hamcrest.core.Is.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import wolox.training.repositories.BookRepository;
+import wolox.training.repositories.UserRepository;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(UserController.class)
+public class UserControllerTest {
```

Comment:
Please add more tests. You could test all CRUD operations, and also the other controller methods such as addBook.